gsa/radix.go

```diff
@@ -1,20 +1,92 @@
 package gsa
 
+import (
+	"fmt"
+	"sort"
+	"strings"
+)
+
 // Sort the string x using a count sort
 func CountSort(x string) string {
-	return ""
+	var sb strings.Builder
+	counts := make(map[byte]int)
+	keys := []int{}
```

Comment:
I'm not sure why you want keys here, though. If you are going to map bytes to counts, then an array of length 256 will get the job done.

```go
  counts := [256]int
```

There are only 256 bytes, so this array will suffice for counting.